app/assets/v2/js/pages/bounty_details.js

```diff
@@ -243,7 +243,8 @@ var pendingChangesWarning = function(issueURL, last_modified_time_remote, now){
             }
             var msg = `<br>This funded issue has recently been updated and while the blockchain syncs it has `+pendingchanges+`.
             Please wait a minute or two for web3 to sync `+this_transaction+`.
-            <br>(Please DO NOT close the browser tab.  This page will automatically refresh as soon as the blockchain is updated.)`
+            <br>(Please DO NOT close the browser tab.  This page will automatically refresh as soon as the blockchain is updated.)
+            <p>QUOTE OF THE DAY: ` + waitingRoomEntertainment() + `</p>`
```

Comment:
the info message is big enough already.. 

might be good to get a designer into this and show off the quote in a beautifully desiigned way on the page itself... 

what does everyone think?